templates/taxi/car_list.html

```diff
@@ -2,16 +2,19 @@
 
 {% block content %}
   <h1>Car list</h1>
+  
+  <a href="{% url 'taxi:car-create' %}" class="btn btn-primary mb-2">Create New Car</a>
+  
   {% if car_list %}
```

Comment:
The condition should check `object_list` instead of `car_list` to be consistent with the context provided by Django's ListView.